src/shared/containers/Refile/RefileErrorsForm.jsx

```diff
@@ -1,53 +1,70 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import axios from  'axios';
+import axios from 'axios';
 import isEmpty from 'lodash/isEmpty';
-import forIn from 'lodash/forIn';
+import { map } from 'lodash';
```

Comment:
Don't really care, but can't we just use the ES6 map function nowadays? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

Reply:
We can! But I just think since Rafael has already using `lodash` in the app, and I kind like its map function...